frame/staking/src/lib.rs

```diff
@@ -692,32 +702,50 @@ pub struct Nominations<T: Config> {
 	pub suppressed: bool,
 }
 
-/// The amount of exposure (to slashing) than an individual nominator has.
-#[derive(PartialEq, Eq, PartialOrd, Ord, Clone, Encode, Decode, RuntimeDebug, TypeInfo)]
-pub struct IndividualExposure<AccountId, Balance: HasCompact> {
-	/// The stash account of the nominator in question.
-	pub who: AccountId,
-	/// Amount of funds exposed.
-	#[codec(compact)]
-	pub value: Balance,
+/// Extended view of Exposure comprising of `ExposureOverview` and a single page of `ExposurePage`.
+///
+/// This is useful where we need to take into account the validator's own stake and total exposure
+/// in consideration, in addition to the individual nominators backing them.
+#[derive(Encode, Decode, RuntimeDebug, TypeInfo, PartialEq, Eq)]
+struct ExposureExt<AccountId, Balance: HasCompact + codec::MaxEncodedLen> {
```

Comment:
Since `ExposureExt` is only used to return `get_paged_exposure`, would `PagedExposure` be a more intuitive name for this struct?

Reply:
Honestly I struggled to name this. Its more of a wrapper struct, so I could potentially rename it to `PagedExposure` as you suggested. Other names that come to my mind is `PagedExposureOuter` or `PagedExposureFacade`.

Reply:
Renamed to `PagedExposure`. Looks more intuitive now I think. Thanks.

```rust
struct PagedExposure<AccountId, Balance> {
	exposure_metadata: PagedExposureMetadata<Balance>,
	exposure_page: ExposurePage<AccountId, Balance>,
}
```